src/main/java/org/dependencytrack/util/VulnerabilityUtil.java

```diff
@@ -201,4 +204,14 @@ public static Set<Map.Entry<Vulnerability.Source, String>> getUniqueAliases(fina
         return uniqueAliases;
     }
 
+    public static void checkInactiveAttributions(final QueryManager qm,
```

Comment:
Nitpick: The name should say `AliasAttribution` instead of just `Attribution`, as we have multiple types of attributions right now.